.travis.yml

```diff
@@ -100,17 +100,6 @@ jobs:
 before_install:
     - |
       # Helper functions
-      run_tests () {
```

Comment:
Could you maybe describe what you did here? This doesn't look like just switching to simple phpunit but a lot more :)

Reply:
Now that there is a global `phpunit.xml.dist` file, we do not need the `run_tests` helper method anymore.

Reply:
Okay, but running only the tests for one bundle would still be useful during development.

Reply:
You still can.

```
vendor/bin/phpunit
# runs all unit tests

vendor/bin/phpunit -c calendar-bundle
# runs only the tests in calendar-bundle/tests
```

Reply:
Ah man, I've confused this with your run shell script 🤦‍♂

Reply:
np